arducam/include/arducam/arducam_driver.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <termios.h>
+#include <thread>
+
+#include "rclcpp/rclcpp.hpp"
+#include "sensor_msgs/image_encodings.hpp"
+#include "sensor_msgs/msg/image.hpp"
+#include "std_msgs/msg/string.hpp"
+
+#include <arducam_config_parser/arducam_config_parser.h>
+#include <arducam_sdk/ArduCamLib.h>
+
+class ArducamDriver : public rclcpp::Node {
```

Comment:
Namespace this `acrobat::arducam`

Reply:
Will need to namespace your component registration the same way (in the cpp and in CMake)